src/fidasim.f90

```diff
@@ -9386,7 +9859,7 @@ end subroutine pnpa_mc
 
 subroutine neutron_f
```

Comment:
You need to handle the situation where the plasma parameters / fields have a phi variable(is not axisymmetric) and the distribution function is axisymmetric.

Reply:
Did you address this?